README.md

```diff
@@ -296,6 +296,50 @@ above. For example, if the mapping file has some stale/incorrect mapping
 entries, they will _not_ be resolved by the Python environment resolver (which
 is usually more accurate).
 
+### Ignoring paths
+
+If you want FawltyDeps to ignore parts of your source tree while it is loooking
+for code, dependency declarations, or Python environments, then you can use the
+`--ignore` option to specify path patterns to ignore, e.g. the following command
+will ignore everything under `tests/`:
+
+```sh
+fawltydeps --ignore tests/
+```
+
+The format of the ignore patterns is the same as used by `.gitignore` files,
+[see here for a full description](https://git-scm.com/docs/gitignore#_pattern_format).
+(Note that some patterns that must be interpreted relative to the location of a
+`.gitignore` file are not (yet) supported by FawltyDeps. Search for "relative"
+in the linked page to see which pattern types are not yet supported.)
+
+When the `--ignore` option is not specified, its default value is `".*"`, which
+matches all paths that start with a dot (`.`), aka. "hidden" paths. In the above
+example, if you want to ignore both hidden paths, and everything under `tests/`,
```

Comment:
Which example are we referring to?

Reply:
`fawltydeps --ignore tests/` on line 307